File: 1-Beginner/1002/bee1002.java
// 1002 - Área do Círculo
import java.io.IOException;
import java.util.Scanner;

class bee1002 {

    public static void main(String[] args) throws IOException{
        Scanner scan = new Scanner(System.in);

        double raio, areaCirculo;
        raio = scan.nextDouble();

        areaCirculo = 3.14159 * Math.pow(raio, 2);

        System.out.printf("A=%.4f\n", areaCirculo);

        scan.close();
    }
}

File: 1-Beginner/1002/bee1002.cpp
// 1002 - Área do Círculo
#include <iostream>
#include <cmath>
#include <iomanip>

int main(void) {

    std::cout<<std::fixed<<std::setprecision(4);

    double raio, areaCirculo;
    std::cin>>raio;

    areaCirculo = 3.14159 * pow(raio, 2);

    
    std::cout<<"A="<<areaCirculo<<std::endl;
}